cmd/masonry/masonry_test.go

```diff
@@ -12,19 +12,24 @@ import (
 
 var usage = `
 Usage:
-  masonry [command]
+  masonry [global-options] COMMAND [command-options]
 
-Available Commands:
+Commands:
   diff        Compliance Diff Gap Analysis
   docs        Create compliance documentation
   export      Export to consolidated output
   get         Install compliance dependencies
   help        Help about any command
+  version     Display version
 
-Flags:
+Options:
```

Comment:
given the ``usage`` output, should this be renamed to ``command-options``?

Reply:
No. It shouldn't.

Reply:
Bah

Reply:
I mean you're right.

default ``help`` behavior vs ``help get``